src/GcsBackup.php

```diff
@@ -7,11 +7,28 @@
 use DateTimeImmutable;
 use Google\Cloud\Storage\StorageClient;
 use Google\Cloud\Storage\StorageObject;
+use GuzzleHttp\Exception\RequestException;
 use Keboola\StorageApi\Client as StorageApi;
 use Psr\Log\LoggerInterface;
+use Retry\BackOff\ExponentialBackOffPolicy;
+use Retry\Policy\CallableRetryPolicy;
+use Retry\RetryProxy;
+use Throwable;
 
 class GcsBackup extends Backup
 {
+    // see https://cloud.google.com/storage/docs/retry-strategy
+    public const RETRY_HTTP_CODES = [
+        408, // Request Timeout
+        429, // Too Many Requests
+        500, // Internal Server Error
+        502, // Bad Gateway
+        503, // Service Unavailable
+        504, // Gateway Timeout
+    ];
+
+    public const RETRY_MAX_TRIES = 3;
```

Comment:
`MAX_RETRIES`?

Reply:
může být -> přejmenuju i v S3 backupu, ať se to jmenuje stejně